index.js

```diff
@@ -29,6 +34,14 @@ Hint #2: you'll need to use the `math` object for parts of this calculation!
 When your math is correct, monthlyRate will equal 1073.64
 */
 
+// let numerator = monthlyInterestRate *  Math.pow(1 + monthlyInterestRate, periods);
+
+// let denominator = Math.pow(1+ monthlyInterestRate, periods)- 1 ;
+
+// let monthlyRate = principal * ( numerator / denominator);
```

Comment:
Almost perfect, instead of getting the long decimal this should print 1073.64 (2 decimal points)
look into the .toFixed() method

Reply:
You can merge this whenever you want